discord/channel.go

```diff
@@ -58,14 +58,14 @@ func (ch Channel) IconURL() string {
 
 type ChannelType uint8
 
-const (
-	GuildText ChannelType = iota
-	DirectMessage
-	GuildVoice
-	GroupDM
-	GuildCategory
-	GuildNews
-	GuildStore
+var (
```

Comment:
Why did we need this again?

Reply:
So we can write something like this:

```go
SomeStruct{
    Type: &GuildText, // type of Type is *ChanneType
}
```

If this were a constant, we couldn't make it a pointer, as it is already immutable.